libioc/Host.py

```diff
@@ -114,6 +115,19 @@ def _init_defaults(
                 zfs=self.zfs
             )
 
+    @property
+    def id(self) -> str:
```

Comment:
This should be `uuid.UUID`, but the uuid module in Python 3.6 is loading terribly slow.

Reply:
I thought we fixed that ourselves somewhere?

Reply:
Yes, but it landed in Python 3.7, which is currently unsupported due to py-libzfs (see the ports Makefile).

Reply:
we could add the shim back in, but only load it on < 3.7